scalding-core/src/main/scala/com/twitter/scalding/typed/TypedPipe.scala

```diff
@@ -143,17 +143,16 @@ object TypedPipe extends Serializable {
       }
   }
 
-  private val identityOrdering: OrderedSerialization[Int] = {
+  private case object IdentityOrdering extends OrderedSerialization[Int] with EquivSerialization[Int] {
     val delegate = BinaryOrdering.ordSer[Int]
-    new OrderedSerialization[Int] {
```

Comment:
I *think* this would have worked with just adding `new OrderedSerialization[Int] with EquivSerialization[Int]` but I think this is fine too.